app/src/main/res/layout/activity_option2.xml

```diff
@@ -42,4 +42,28 @@
         android:textAppearance="@style/text_title"
         android:background="@color/colorAccent"/>
 
+    <RelativeLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
+
+        <Button
+            android:id="@+id/btn_back_2"
+            android:layout_width="160dp"
+            android:layout_height="wrap_content"
+            android:text="Back"
+            android:onClick="onClick"
```

Comment:
Do we still need this?

Reply:
Not that point. I was focusing about onClick which we might not need it since we used setOnClickListener.